tests/test_workspace_churn_up_arrow_lag.py

```
@@ -179,10 +179,10 @@ def resolve_target_socket() -> str:
     socket_path = os.environ.get("CMUX_SOCKET_PATH")
     if not socket_path:
         raise cmuxError(
-            "CMUX_SOCKET_PATH is required. Point it to a tagged dev socket (for example /tmp/cmux-debug-<tag>.sock)."
+            "CMUX_SOCKET_PATH is required. Point it to a tagged DEV socket."
         )
     base = os.path.basename(socket_path)
-    if not ALLOW_MAIN_SOCKET and base in {"cmux.sock", "cmux-debug.sock"}:
+    if not ALLOW_MAIN_SOCKET and base in {"cmux.sock", "cmux-debug.sock", "com.cmuxterm.app.sock"}:
         raise cmuxError(
             f"Refusing to run against main socket '{socket_path}'. Set CMUX_SOCKET_PATH to a tagged dev instance."
         )
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)